kafka-connect/kafka-connect-runtime/LICENSE

```diff
@@ -0,0 +1,1970 @@
+
+Group: io.projectreactor  Name: reactor-core  Version: 3.4.38
+Project URL (from POM): https://github.com/reactor/reactor-core
+License (from POM): Apache License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.projectreactor.netty  Name: reactor-netty-core  Version: 1.0.45
+Project URL (from POM): https://github.com/reactor/reactor-netty
+License (from POM): The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: io.projectreactor.netty  Name: reactor-netty-http  Version: 1.0.45
+Project URL (from POM): https://github.com/reactor/reactor-netty
+License (from POM): The Apache Software License, Version 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+
+--------------------------------------------------------------------------------
+
+Group: jakarta.activation  Name: jakarta.activation-api  Version: 1.2.1
+Project URL (from manifest): https://www.eclipse.org
+License (from POM): EDL 1.0 - http://www.eclipse.org/org/documents/edl-v10.php
+License (from POM): Eclipse Public License v. 2.0 - https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt
+License (from POM): GNU General Public License, version 2 with the GNU Classpath Exception - https://www.gnu.org/software/classpath/license.html
+
+--------------------------------------------------------------------------------
+
+Group: jakarta.activation  Name: jakarta.activation-api  Version: 1.2.2
+Project URL (from manifest): https://www.eclipse.org
+License (from POM): EDL 1.0 - http://www.eclipse.org/org/documents/edl-v10.php
+License (from POM): Eclipse Public License v. 2.0 - https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt
+License (from POM): GNU General Public License, version 2 with the GNU Classpath Exception - https://www.gnu.org/software/classpath/license.html
+
+--------------------------------------------------------------------------------
+
+Group: jakarta.xml.bind  Name: jakarta.xml.bind-api  Version: 2.3.3
+Project URL (from manifest): https://www.eclipse.org
+License (from POM): Eclipse Distribution License - v 1.0 - http://www.eclipse.org/org/documents/edl-v10.php
+License (from POM): Eclipse Public License v. 2.0 - https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.txt
+License (from POM): GNU General Public License, version 2 with the GNU Classpath Exception - https://www.gnu.org/software/classpath/license.html
+
+--------------------------------------------------------------------------------
+
+Group: javax.annotation  Name: javax.annotation-api  Version: 1.3.2
+Project URL (from manifest): https://javaee.github.io/glassfish
+Project URL (from POM): http://jcp.org/en/jsr/detail?id=250
+License (from POM): CDDL + GPLv2 with classpath exception - https://github.com/javaee/javax.annotation/blob/master/LICENSE
```

Comment:
Is it ok to have this GPL? 

https://www.apache.org/legal/resolved.html#category-x

cc: @Fokko

Reply:
Yes, that's the special exception:

> Special exceptions to the GNU GPL (e.g. GNU Classpath) unless otherwise permitted elsewhere on this page.